build.sbt

```
@@ -103,7 +106,7 @@ lazy val circeAkkaSerializer = (projectMatrix in file("circe-akka-serializer"))
       virtualAxes.value.collectFirst { case x: ScalaVersionAxis => x.value }
     })
   .settings(
-    resolvers += Resolver.sonatypeRepo("releases"),
+    resolvers += Resolver.sonatypeRepo("releases"), // TODO (#202): update to sonatypeOssRepos
```

Maybe just fix on this PR? or will it require more changes than just
```suggestion
    resolvers += Resolver.sonatypeOssRepos("releases"),
```

---

Looks like something more - I tried this, but the sonatypeOssRepos method works differently